pages/mes-jeunes/[jeune_id]/actions/[action_id].tsx

```diff
@@ -173,6 +180,15 @@ function PageAction({ action, jeune, commentaires }: PageActionProps) {
         />
       )}
 
+      {lectureSeule && (
+        <div className='mb-6'>
+          <InformationMessage label='Vous êtes en lecture seule'>
+            Vous pouvez uniquement lire la fiche de ce bénéficiaire car il ne
```

Comment:
```suggestion
Vous pouvez uniquement lire le détail de l’action de ce bénéficiaire car il ne fait pas partie de votre portefeuille.
```
cf. https://www.notion.so/fabnummas/Conseiller-Milo-non-r-f-rent-peut-consulter-le-d-tail-des-actions-du-jeune-a863b39c34424a0b80995437ef5a0b2e